native/src/routes/wohnen/containers/WohnenOfferContainer.js

```diff
@@ -14,23 +14,24 @@ import type { NavigationStackProp } from 'react-navigation-stack'
 import FailureContainer from '../../../modules/error/containers/FailureContainer'
```

Comment:
This would not have been necessary, we generally don't want to put any effort in wohnen anymore

Reply:
it was more or less just copy pasting stuff. But if you think it's better to remove it to avoid future errors resulting from the "new" code I will do so :)

Reply:
No, the work is already done :D Putting even more work into it to remove it again would not make sense imo :grin: